packages/exit-app/src/utils/explorers.ts

```diff
@@ -138,6 +138,14 @@ export const EXPLORERS_CONFIG: Record<NETWORK, ExplorerData> = {
     verifyContractUrl: "https://sepolia.lineascan.build/verifyContract",
     explorerApiKey: process.env.REACT_APP_LINEASCAN_KEY ?? '',
   },
+  [NETWORK.PLASMA_TESTNET]: {
+    networkExplorerName: "Plasmascan",
+    networkExplorerUrl: "https://testnet.plasmascan.to/",
+    networkExplorerApiUrl: "https://testnet.plasmascan.to/api",
```

Comment:
```suggestion
    networkExplorerApiUrl: "https://api.routescan.io/v2/network/testnet/evm/9746_5/etherscan/api",
```
I think it better to use this one from the docs https://testnet.plasmascan.to/documentation/api/etherscan-like/accounts